src/explore-education-statistics-common/src/modules/table-tool/components/__tests__/PublicationForm.test.tsx

```diff
@@ -16,157 +16,93 @@ describe('PublicationForm', () => {
-              isSuperseded: false,
-            },
-          ],
+          id: 'publication-2',
+          title: 'Publication 2 find me',
+          slug: 'publication-slug-2',
+          isSuperseded: false,
         },
       ],
     },
     {
       id: 'theme-2',
       title: 'Theme 2',
       summary: '',
-      topics: [
+      publications: [
         {
-          id: 'topic-3',
-          title: 'Topic 3',
-          summary: '',
-          publications: [
-            {
-              id: 'publication-3',
-              title: 'Publication 3',
-              slug: 'publication-slug-3',
-              isSuperseded: false,
-            },
-          ],
+          id: 'publication-3',
+          title: 'Publication 3',
+          slug: 'publication-slug-3',
+          isSuperseded: false,
         },
       ],
     },
     {
       id: 'theme-3',
       title: 'Theme 3',
       summary: '',
-      topics: [
+      publications: [
+        {
+          id: 'publication-4',
+          title: 'Publication 4 find me',
+          slug: 'publication-slug-4',
+          isSuperseded: false,
+        },
         {
-          id: 'topic-4',
-          title: 'Topic 4',
-          summary: '',
-          publications: [
-            {
-              id: 'publication-4',
-              title: 'Publication 4 find me',
-              slug: 'publication-slug-4',
-              isSuperseded: false,
-            },
-          ],
+          id: 'publication-5',
+          title: 'Publication 5',
+          slug: 'publication-slug-5',
+          isSuperseded: false,
         },
         {
-          id: 'topic-5',
-          title: 'Topic 5',
-          summary: '',
-          publications: [
-            {
-              id: 'publication-5',
-              title: 'Publication 5',
-              slug: 'publication-slug-5',
-              isSuperseded: false,
-            },
-          ],
+          id: 'publication-6',
+          title: 'Publication 6',
+          slug: 'publication-slug-6',
+          isSuperseded: false,
         },
         {
-          id: 'topic-6',
-          title: 'Topic 6',
-          summary: '',
-          publications: [
-            {
-              id: 'publication-6',
-              title: 'Publication 6',
-              slug: 'publication-slug-6',
-              isSuperseded: false,
-            },
-          ],
+          id: 'publication-7',
+          title: 'Publication 7',
+          slug: 'publication-slug-7',
+          isSuperseded: false,
         },
         {
-          id: 'topic-7',
-          title: 'Topic 7',
-          summary: '',
-          publications: [
-            {
-              id: 'publication-7',
-              title: 'Publication 7',
-              slug: 'publication-slug-7',
-              isSuperseded: false,
-            },
-          ],
+          id: 'publication-7',
+          title: 'Publication 7',
+          slug: 'publication-slug-7',
+          isSuperseded: false,
```

Comment:
just curious about why we have a duplicate publication-7 listed?

Reply:
Cut/paste duplicate error, I'll remove.